Example/Controllers/DisplayController.swift

```diff
@@ -36,4 +40,21 @@ class DisplayController: UIViewController {
             break
         }
     }
+
+    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+        DisplayController.miniAppSupportedOrientation = self.miniAppDisplayDelegate?.getSupportedOrientation() ?? .all
+        return DisplayController.miniAppSupportedOrientation
+    }
+}
+
+extension DisplayController: UINavigationControllerDelegate {
+    public func navigationControllerSupportedInterfaceOrientations(_ navigationController: UINavigationController) -> UIInterfaceOrientationMask {
+        return navigationController.topViewController?.supportedInterfaceOrientations ?? .all
+    }
+}
+
+extension AVPlayerViewController {
```

Comment:
This seems a little complicated that we have to ask the Host App to override the supported orientation interfaces for their view controller, navigation controller, and the avplayer. But I guess there isn't really any other way of doing this 😣

Reply:
Yes, that's the sad part here. 

1. `override var supportedInterfaceOrientations: UIInterfaceOrientationMask`
 This is to override the supported orientation for the view controller i.e the display controller which the host app would use to display the mini-app. If the mini-app wants to lock the orientation for the viewcontroller(not just the video), they can do it only if the host app override this.
2. `public func navigationControllerSupportedInterfaceOrientations`
 DisplayController is pushed in the navigation controller and without this override, Viewcontroller is not locking the orientation. This is little strange for me too, when I tested in a small sample app, orientation lock is working fine without this override. I will verify it one more time.
3. `AVPlayerViewController - > override open var supportedInterfaceOrientations`
This is to override the orientation of the default media player. The host app can still skip this one, this is not mandatory. If the host app didn't override this, then the video wouldn't rotate automatically to landscape mode when the lock is set. User has to manually rotate the screen to lock in the landscape mode.

Reply:
Thanks, this is probably okay. About `AVPlayerViewController`, this is specifically what the orientation feature is needed for - to automatically set videos to landscape mode. 

Seems this will be confusing to the Host App, but it should be okay if we clearly document in the user guide how to handle orientation and what each override is for.

Reply:
Sure, I will add it in the documentation. I just noticed now that if the host app didn't override `AVPlayerViewController - > override open var supportedInterfaceOrientations` then the orientation lock is not working. This is because the Video player opened from the mini-app. I guess the host app need to implement this for sure, I will update all these things in the documentation once the PR is merged.